core/Plugin/Manager.php

```diff
@@ -1191,6 +1191,8 @@ private function executePluginDeactivate($pluginName)
      */
     private function unloadPluginFromMemory($pluginName)
     {
+        $this->unloadPlugin($pluginName);
+
```

Comment:
`Plugin::uninstall` is more or less meant as a hook where plugins can remove eg created files, DB tables/data etc in case the plugin gets uninstalled.